templates/auth/register/main.scss

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2022  Aravinth Manivannan <realaravinth@batsense.net>
```

Comment:
Minor nitpick: wrong copyright header, can you please change it?

Reply:
Sorry for the late fix, you know why :/